zproject/settings.py

```diff
@@ -157,6 +157,7 @@ def get_secret(key):
                     'REGISTER_LINK_DISABLED': False,
                     'LOGIN_LINK_DISABLED': False,
                     'ABOUT_LINK_DISABLED': False,
+                    'FIND_TEAM_LINK_DISABLED': False,
```

Comment:
This probably should default to True, since you only need this on a multi-realm server (which isn't the default).